charms/katib-ui/pyproject.toml

```diff
@@ -72,10 +74,23 @@ isort = "^5.13.2"
 pep8-naming = "^0.14.1"
 pyproject-flake8 = "^7.0.0"
 
+[tool.poetry.group.unit]
+optional = true
+
+[tool.poetry.group.unit.dependencies]
+coverage = "^7.6.1"
+ops = "^2.17.1"
```

Comment:
nitpick: `ops` is already coming from the charm's dependencies, shall we have only dependencies to be added on top of the charm ones, here, for unit testing, given that we install both dependency groups when running unit tests?

Reply:
Fair enough!